src/index.html

```diff
@@ -2,15 +2,33 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="/src/index.html" defer></script>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input
+        data-qa="keypress"
+        class="search-form__input search-form__input_big"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
+
+    <form class="search-form search-form_small" data-qa="small">
+      <label class="search-form__label search-form__label_small">
+        <input
+        data-qa="hover"
+        class="search-form__input search-form__input_small"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
code is not aligned
check other places 
```suggestion
        <input
          data-qa="hover"
          class="search-form__input search-form__input_small"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```